Blackjack.py

```diff
@@ -0,0 +1,282 @@
+        playing = True
+
+        take_bet(player_chips)
+        dealer_chips.bet = player_chips.bet
+
+        the_deck.shuffle()
+
+        for x in range(0, 2):
+            player.add_card(the_deck.deal())
+
+        for y in range(0, 2):
+            dealer.add_card(the_deck.deal())
+
+        show_some(player, dealer)
+
+        while playing and player.player_value != 21:  # single game until either the player or dealer wins the bet
+
+            # player turn
+
+            hit_or_stand(the_deck, player)
+
+            show_some(player, dealer)
+
+            if player.player_value >= 21:
+                break
+
+                # player bust haha
+        if player.player_value > 21:
+            player_busts(player, player_chips)
+
+        if player.player_value == 21:
+            print("MF Blackjack!!!!!, Dealer Loses")
+            player_wins(player, player_chips)
+            dealer_chips.lose_bet()
+
+        # Player didn't bust, lets play dealers hand
+        if player.player_value < 21:
+            print("\nTime to show all the cards\n")
+            show_all(player, dealer)
+
+            print("\nDealer's Turn")
+            while dealer.player_value < 17:
+                hit(the_deck, dealer)
+
+            if dealer.player_value > 21:
+                dealer_busts(dealer, dealer_chips)
+                player_chips.win_bet()
+
+            elif dealer.player_value == 21:
+                dealer_wins(dealer, player_chips)
+                player_chips.lose_bet()
+
+            elif player.player_value > dealer.player_value:
+                player_wins(player, player_chips)
+                dealer_chips.lose_bet()
+
+            elif dealer.player_value > player.player_value:
+                dealer_wins(dealer, dealer_chips)
+                player_chips.lose_bet()
+
+            elif player.player_value == dealer.player_value:
+                push()
+
+        print("\nTime to show all the cards AGAIN!\n")
+        show_all(player, dealer)
+
+        print(f"\nPlayer has {player_chips.total} chips\nDealer has {dealer_chips.total} chips")
+
+        while True:
+            answer = input("Does player want to go another round? (Y or N): ")
+            if answer.lower() == "y" or answer.lower() == "n":
+                break
+            else:
+                print('please only input either a "Y" or a "N"')
+                continue
+
+        if answer.lower() == "n" or player_chips.total == 0 or dealer_chips.total == 0:
+            print("Sorry you are a sore loser")
+            break
+
+        player.all_cards = []
+        dealer.all_cards = []
+        round += 1
+        #This project originated in Jupyter and therefore the clear_output() command was used
+        #This will be a seperate commit on the Blackjack branch
+        #And one more push from my local machine to Blackjack branch Github
+        #clear_output()

```

Comment:
Forgot to mention, this isn't really needed. You can run your program outside of Jupyter by running:

`python3 Blackjack.py`
Then you can type `clear` in the terminal when the program is done if you want to clear the screen (assuming bash or bash-like shell for the clear command).

I also recommend lower case for .py files. I think that's supposed to be standard PEP8 guidelines.

Reply:
Ah thanks, haha wish I would have saw this before creating the new files. Oh well, I can do it in the branch.

Reply:
Ah thanks, haha wish I would have saw this before creating the new files. Oh well, I can do it in the branch.